assets/src/setup/pages/index.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { TechnicalBackground } from './technical-background';
+import { Goals } from './goals';
+import { TemplateMode } from './template-mode';
+import { ChooseReaderTheme } from './choose-reader-theme';
+import { SiteConfigurationSummary } from './site-configuration-summary';
+import { Save } from './save';
+import { SiteScan } from './site-scan';
+
+/**
+ * Settings for the pages in the application.
+ */
+export const PAGES = [
```

Comment:
The titles here should where possible be the same as used in the corresponding `PageComponent` to avoid re-translating (e.g. Title Casing)